package.json

```diff
@@ -13,7 +13,7 @@
     "test": "jest --verbose",
     "lint": "prettier --check .",
     "format": "prettier --write .",
-    "postinstall": "husky install",
+    "_postinstall": "husky install",
```

Comment:
Any reason for the added underscore here?

Reply:
Ah, I see that this is something that Husky does automatically (noticed the same thing in my local version too)